datafusion/sql/src/planner.rs

```diff
@@ -154,6 +156,7 @@ impl PlannerContext {
             ctes: HashMap::new(),
             outer_query_schema: None,
             outer_from_schema: None,
+            table_schema: None,
```

Comment:
a query may affect multiple tables, so ``main_table_schema``?

Reply:
Is it possible that there are complex queries which define multiple table schemas at once? 🤔 If so, I think we could work on another PR focused on supporting that type of query.

Reply:
Maybe we can name it `create_table_schema`  to reflect that it is used for `CREATE TABLE` processing?

Reply:
> Maybe we can name it `create_table_schema` to reflect that it is used for `CREATE TABLE` processing?

that would work too